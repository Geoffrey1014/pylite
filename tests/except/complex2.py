def main():
    try:
        print 'main start'
        0/0
        print 'main end'
    except:
        print 'except start'
        raise IOError()
        print 'except end'
    finally:
        print 'finally' 

print 'global start'
main()
print 'global end'
