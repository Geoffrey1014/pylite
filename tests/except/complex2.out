global start
main start
except start
finally
IOError:
  At complex2.py, line 8, in main()
  At complex2.py, line 14, in __init__()
